Remove bracket rotations in favor of <x, y, z>. Addresses #757. (#888)
thewca/wca-regulations: wca-guidelines.md

- E2c2+) [EXAMPLE] Examples that are not considered a single unambiguous move sequence: multiple solution sequences without exactly one clearly indicated solution, a solution with ambiguous symbols (e.g. a letter that looks similar to "B" and "R" but is not clearly one or the other), a solution with stars or arrows indicating moves to be performed out of written order (e.g. "insertions" that are not written in-line with the rest of the move sequence, "pre-moves" that are not incorporated into the move sequence using valid notation), a move sequence that is not written out roughly as a series of lines in reading order, any arrangement of symbols that cannot be interpreted as an unambiguous move sequence. Exceptions to the examples above: small typographic corrections (e.g. blacking out moves and writing moves above that are clearly meant to replace them, moves written in subscript or superscript to fit between other moves).

> - E2c2++) [REMINDER] Bracket notations (e.g. [r] or [u2]) were previously allowed for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).

Author: ```suggestion
- E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
```

> - E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).

Reviewer: double "based"!

Reviewer: ```suggestion
- E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
```

Author: 🤦‍♀️Thanks!
